build-system/compile/compile-wrappers.js

```diff
@@ -150,4 +150,15 @@ function bento(name, version, latest, isModule, loadPriority) {
 
 exports.bento = bento;
 
+/**
+ * Wraps the CDN build with a defineElement function for the
+ * user to invoke
+ */
+exports.web =
+  'export function defineElement(){(function(AMP,_){<%= contents %>})' +
+  '({registerElement:function(n,b,s){if(s){document.head.appendChild(' +
+  "document.createElement('style')).textContent=s;}" +
+  "customElements.define(n.replace(/^amp-/,'bento-')," +
+  'b.CustomElement(b));},})}';
+
```

Comment:
`web` might not be the best name. I would go for `defineBentoElement` or `bentoFunction` or anything else with `bento` in the name.

And nit: It would be nice to use `removeWhitespace`. `<%= contents %>` may be minified and ignored down the line, but I think it would be fine to revert it on a second `replace` as I'm doing [here](https://github.com/ampproject/amphtml/pull/36174/files#diff-11445f696ba7611042d7c4d6d2691c2e9571fda8d091bd4055037111335ec9ccR3-R7)